tests/scenic/test_scenic.py

```diff
@@ -179,3 +179,56 @@ def test_driving_dynamic(pathToLocalFile):
                                   server_class=ScenicServer,
                                   server_options=server_options)
     falsifier.run_falsifier()
+
+def test_driving_dynamic_behavior(pathToLocalFile):
+    path = pathToLocalFile('scenic_driving_behavior.scenic')
+    sampler = ScenicSampler.fromScenario(
+        path,
+        model='scenic.simulators.newtonian.driving_model',
+        params=dict(render=False),
+        mode2D=True,
+        maxSteps=2
+    )
+    falsifier_params = DotMap(
+        n_iters=3,
+        save_error_table=False,
+        save_safe_table=False,
+    )
+    server_options = DotMap(maxSteps=2, verbosity=3)
+    falsifier = generic_falsifier(sampler=sampler,
+                                  falsifier_params=falsifier_params,
+                                  server_class=ScenicServer,
+                                  server_options=server_options)
+    falsifier.run_falsifier()
```

Comment:
Let's either extend this test or add a new one where you actually check that the time series is being sampled separately at each time step.